lib/utils/domFns.js

```diff
@@ -109,13 +109,28 @@ export function offsetXYFromParent(evt: {clientX: number, clientY: number}, offs
   return {x, y};
 }
 
-export function createCSSTransform({x, y}: {x: number, y: number}): Object {
-  // Replace unitless items with px
-  return {[browserPrefixToKey('transform', browserPrefix)]: 'translate(' + x + 'px,' + y + 'px)'};
+export function createCSSTransform({x, y}: ControlPosition, positionOffset: PositionOffsetControlPosition): Object {
+  let translation;
```

Comment:
```suggestion
  let translation = `translate(${x}px,${y}px)`;
```

Reply:
Could just define it here and prepend the default if present to save a few lines.

Reply:
Done